services/vision/mlvision/ml_model.go

```diff
@@ -58,7 +58,9 @@ func init() {
 
 // MLModelConfig specifies the parameters needed to turn an ML model into a vision Model.
 type MLModelConfig struct {
-	ModelName string `json:"mlmodel_name"`
+	ModelName        string            `json:"mlmodel_name"`
+	RemapInputNames  map[string]string `json:"remap_input_names"`
+	RemapOutputNames map[string]string `json:"remap_output_names"`
```

Comment:
Very much into the name remapping--it's important.  I'm not sure that we need two separate optional params for this though. I could be convinced otherwise but, it would simplify things to just have the one remapNames (or remapTensorNames) map. Is there a benefit to storing them separately?

Reply:
The only thing I was thinking about if for some reason an input tensor and output tensor were named the same thing. So like, if the first input tensor was called "tensor0" and the first output tensor was also called "tensor0". I don't know if that's possible in any framework but wanted to take that into account

Reply:
Ooooh you know what, that's valid. Seems like the exact kinda bs that would _actually_ happen. Convinced.